### DISCUSSION TITLE
GitHub Actions is currently disabled for your account.

### DISCUSSION THREAD

**Discussion by CeroWalker** [community/community#112437] [Actions] [answered]
### Select Topic Area

Question

### Body

How can I reenable Github Actions?

**Comment by AidenJamesYt** [comment]
For a repository, here are the steps:

Access Repository Settings:

Navigate to the repository on GitHub.
Click on the "Settings" tab near the top-right corner of the repository page.
Navigate to Actions Settings:

In the Settings sidebar, click on the "Actions" option. This will take you to the Actions settings page for the repository.
Enable Actions:

On the Actions settings page, you should see a section labeled "Actions Permissions".
If GitHub Actions is disabled, you'll see an option to enable it. Click on the button or link to enable GitHub Actions.

**Reply by CeroWalker** [reply]
It's disabled in my account, not in the repo.

**Reply by AidenJamesYt** [reply]
hm

**Reply by AidenJamesYt** [reply]
weird

**Comment by cirosantilli** [comment, marked as answer]
Related: https://github.com/orgs/community/discussions/19015
